sdks/python/setup.py

```diff
@@ -148,35 +148,6 @@ def run(self):
     return original_cmd
 
 
-def generate_common_urns():
-  src = os.path.join(
-      os.path.dirname(__file__),
-      '../../'
-      'model/pipeline/src/main/resources/org/apache/beam/model/common_urns.md')
-  out = os.path.join(
-    os.path.dirname(__file__),
-    'apache_beam/portability/common_urns.py')
```

Comment:
If we're not auto-generating this, we need to check it in (or something else that extracts all the names from the annotations).

Reply:
Oh snap. I *did* check it in, but it is ignored in .gitignore (forgot to remove this) and cleaned by "mvn clean" (remembered to remove from there when it was too late) :-/ Guess I'll have to redo it.

Reply:
We should not just check this in, that would be a regression. The key point is that there's a common source of truth (namely the protos).

Reply:
(I mean, checking in the auto-generated file. I assume what you did is rewrite this file to extract the proto annotations.)